src/forms/services/form-submission.service.ts

```
@@ -3025,6 +3024,12 @@
         ? user.updatedAt.toISOString()
         : new Date().toISOString(),
     };
+
+    // Debug logging
+    LoggerUtil.warn(`buildUserDocumentForElasticsearch returning ${applications.length} applications for user: ${userId}`);
+    if (applications.length > 0) {
+      LoggerUtil.warn(`First application: ${JSON.stringify(applications[0], null, 2)}`);
+    }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)